### DISCUSSION TITLE
howto convert deprecated WithDailyTimeIntervalSchedule into something similar in newer version of quartz?

### DISCUSSION THREAD

**Discussion by markat1** [quartznet/quartznet#2564] [Q&A] [answered]
So the following now gives me a depecration warning that says something like `"TriggerExtensions.WithDailyTimeIntervalSchedule(ITriggerConfigurator, Action<DailyTimeIntervalScheduleBuilder>?) is obsolete: 'Prefer overload taking explicit interval and interval unit before configure action"`

Which basically means I should refactor the following.

So how do I convert this into something that's not deperacted?

```
   .WithDailyTimeIntervalSchedule
    (s =>
        s.WithIntervalInHours(24)
       .OnMondayThroughFriday()
       .StartingDailyAt(TimeOfDay.HourAndMinuteOfDay(6, 0))
       .WithIntervalInSeconds(5)
       .EndingDailyAt(TimeOfDay.HourAndMinuteOfDay(18, 0))
     ));
```

**Comment by lahma** [comment, marked as answer]
See #2160

**Reply by markat1** [reply]
thanks @lahma, that worked! not sure I really understand the deprecation due to the small difference, but okay :)
